Brixcer_Box/include/LedControl.h

```diff
@@ -1,14 +1,25 @@
+#include <Adafruit_NeoPixel.h>
+
 class LedControl {
 
     // Length of array specifying RGB values
     static const int COLORS = 3;
 
-    int ledPin;
-    int volume;
-    int color_matrix[COLORS];
+    // Number of LEDs per ring
+    static const int LEDS_PER_RING = 12;
+
+    // Number of LEDs dedicated to application color
+    static const int APPLICATION_LEDS = 4;
```

Comment:
Dumb question since I am mobile. Is the ring set to have 2 bottom LEDs or 1? If it's 1 this should be odd.